### PR TITLE
Old man yaoi 2, but this time purple

### PR DESCRIPTION
**Pull request by Draggeru**
Adds the clothing and hair required for Finster, only able to be spawned via game panel

finstersmol_2

### CODE REVIEW

File: maplestation_modules/story_content/noble_equipment/code/nobleclothing.dm [hunk 1]
```diff
@@ -160,3 +160,60 @@
...
+	righthand_file = 'maplestation_modules/story_content/noble_equipment/icons/noble_rhand.dmi'
+	icon_state = "penrosesuit"
+	inhand_icon_state = "penrosesuit"
+	supports_variations_flags = CLOTHING_NO_VARIATION
+	can_adjust = TRUE
+	resistance_flags = FIRE_PROOF|ACID_PROOF
+
+/obj/item/clothing/suit/toggle/noble/penrose
+	name = "ancient dinner coat"
+	desc = "A luxurious coat made from an unidentifiable material. The jem on the coat begs you to step closer."
```
Comment (CRITAWAKETS) [inline maplestation_modules/story_content/noble_equipment/code/nobleclothing.dm:179]:
```suggestion
	desc = "A luxurious coat made from an unidentifiable material. The gem on the coat begs you to step closer."
```
unless this is actually meant to be Jem and the Holograms in which case strange reference but ok

File: maplestation_modules/story_content/noble_equipment/code/nobleclothing.dm [hunk 2]
```diff
@@ -160,3 +160,60 @@
...
+	worn_icon = 'maplestation_modules/story_content/noble_equipment/icons/noble_worn.dmi'
+	lefthand_file = 'maplestation_modules/story_content/noble_equipment/icons/noble_lhand.dmi'
+	righthand_file = 'maplestation_modules/story_content/noble_equipment/icons/noble_rhand.dmi'
+	icon_state = "penrosegloves"
+	inhand_icon_state = "penrosegloves"
+	resistance_flags = FIRE_PROOF|ACID_PROOF
+
+/obj/item/clothing/shoes/jackboots/penrose
+	name = "ancient boots"
+	desc = "An old pair of boots made from an nidentifable material. These boots have taken their fair share of steps."
```
Comment (CRITAWAKETS) [inline maplestation_modules/story_content/noble_equipment/code/nobleclothing.dm:201]:
```suggestion
	desc = "An old pair of boots made from an unidentifable material. These boots have taken their fair share of steps."
```

**Review by CRITAWAKETS — CHANGES_REQUESTED** [review]
You forgor to put these in the dme. meaning these won't appear ingame.

File: maplestation_modules/story_content/noble_equipment/code/nobleclothing.dm (hunk 1 above)
Comment (Draggeru) [reply maplestation_modules/story_content/noble_equipment/code/nobleclothing.dm:179]: No I just can't read while sleep deprived

**Comment by Draggeru** [comment]
> You forgor to put these in the dme. meaning these won't appear ingame.

Unironically works on my machine, so I don't know how to fix this

**Comment by CRITAWAKETS** [comment]
likely you just forgot to push maplestation.dme or something's been fucky with the git

**Comment by Draggeru** [comment]
Explain how do,,,

**Comment by CRITAWAKETS** [comment]
ok so it turns out im dumb and this should werk i somehow didnt notice it wasnt in a new file

**Comment by doshMobile** [comment]
FINSTER SPOTTED

**Pull request merged**